tests/role/test_role_store.cairo

```diff
@@ -42,6 +42,25 @@ fn given_normal_conditions_when_has_role_after_revoke_then_works() {
     // Check that the account address does not have the admin role.
     assert(!role_store.has_role(account_1(), ROLE_ADMIN), 'Invalid role');
 }
+#[test]
+#[should_panic]
```

Comment:
Could you precise the expected panic error

example: `#[should_panic(expected: ('adl_not_enabled',))]`